### DISCUSSION TITLE
Getting 400 Bad Request (bad accept header)

### DISCUSSION THREAD

**Discussion by jonlamb-gh** [keygen-sh/community#74] [Issue]
One of the ways we're using keygen is to host a debian apt repo for our software packages.
Recently we started getting `400 Bad Request`'s when attempting to fetch the repository files (`apt-get update`, etc):
```
Err:10 https://get.keygen.sh/auxon-io/debian jammy InRelease                                                                           
  400  Bad Request [IP: 3.232.242.170 443]
```

Looking at the request (`apt-get -oDebug::Acquire::https=1 update`), it appears to indicate that `text/*` accept request-header field is no longer supported.
```
GET /auxon-io/debian/dists/jammy/InRelease HTTP/1.1
Host: get.keygen.sh
Cache-Control: max-age=0
Accept: text/*
User-Agent: Debian APT-HTTP/1.3 (2.4.9)
```

I can reproduce with curl:
```
curl -v --header "Accept: text/*" -A "Debian APT-HTTP/1.3 (2.4.9)" https://get.keygen.sh/auxon-io/debian/dists/jammy/InRelease
...
{"meta":{"id":"bf209837-8116-4015-a989-9fb0d7a4d308"},"errors":[{"title":"Bad request","detail":"The content type of the request is not supported (check accept header)","code":"ACCEPT_INVALID"}]}
```

This setup has previously been working for quite some time, did something change on keygen's side (using `*/*` works as expected FWIW)?

**Comment by ezekg** [comment]
Sorry about the trouble here. Looks like we introduced this bug a few days ago. We just pushed out a fix.

Let me know if the issue is resolved. I did a test request and it looks to be fixed.

**Reply by jonlamb-gh** [reply]
Thanks for the quick fix @ezekg, things are working now.
